frontend/views/containers/contributions/IncomeDetails.vue

```diff
@@ -197,7 +194,7 @@ export default {
         [L('This field is required')]: required,
         [L("It seems your income is not lower than the group's mincome.")]: function (value) {
```

Comment:
Boyscoutting:

`"It seems your income is not lower than the group's mincome."` => `"Income must be lower than group mincome"`

Reply:
Can we write a complete sentence so it doesn't sound so robotic? Maybe `Your income must be lower than the group mincome.`

Reply:
Agreed, that sound even better :)